### PR TITLE
(SIMP-9699) Remove SIMP specific defaults

### PR DESCRIPTION
**Pull request by jeannegreulich**
- Removed the configuration for simp accounts default instance.
  This was moved to a new profile type module, simp_ds389.

SIMP-9699 #comment updates to ds389.

### CODE REVIEW

File: spec/acceptance/suites/default/00_default_spec.rb [hunk 1]
```diff
@@ -65,39 +72,33 @@
...
       end
 
       it 'fails when logging in with forced encryption' do
-        expect { on(host, %(ldapsearch -ZZ -x -y "#{root_dn_password_file}" -D "cn=Directory_Manager" -h `hostname -f` -b "cn=tasks,cn=config")) }.to raise_error(Beaker::Host::CommandFailure)
+        expect { on(host, %(ldapsearch -ZZ -x -w "#{rootpasswd}" -D "#{root_dn}" -h `hostname -f` -b "cn=tasks,cn=config")) }.to raise_error(Beaker::Host::CommandFailure)
       end
     end
 
     context 'with an instance to delete' do
+      let(:root_dn_password_file) do
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/default/00_default_spec.rb:95]: In other parts of the test, you use the password directly, in lieu of the password file.  Why did you leave use of the password file here?

File: spec/acceptance/suites/default/10_pki_spec.rb [hunk 2]
```diff
@@ -10,27 +10,31 @@
...
+      let(:rootpasswd) { 'password'}
+      let(:root_dn) { 'cn=Directory_Manager' }
+      let(:bootstrapldif) { ERB.new(File.read(File.expand_path('files/bootstrap.ldif.erb',File.dirname(__FILE__)))).result(binding) }
+      let(:fqdn) {  fact_on(host, 'fqdn').strip }
+      let(:certdir) {'/etc/pki/simp-testing/pki'}
 
-    fqdn = fact_on(host, 'fqdn').strip
-
-    context 'when enabling PKI in the default instance' do
+      let(:root_dn_password_file) do
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/default/10_pki_spec.rb:22]: I don't think this is used.  Please remove.

**Review by lnemsick-simp — CHANGES_REQUESTED** [review]
* Minor test nits
* Please consider addressing SIMP-9723 in this PR.

File: spec/acceptance/suites/default/00_default_spec.rb (hunk 1 above)
Comment (jeannegreulich) [reply spec/acceptance/suites/default/00_default_spec.rb:95]: In that instance it autogenerates the password using passgen and I don't have the password handy.  I can change it to not auto-generate it.  Do you think that is necessary?

**Review by trevor-vaughan — COMMENTED** [review]
:eye: :+1:

File: spec/acceptance/suites/default/10_pki_spec.rb (hunk 2 above)
Comment (jeannegreulich) [reply spec/acceptance/suites/default/10_pki_spec.rb]: I removed it.

File: .github/workflows/pr_acceptance.yml [hunk 3]
```diff
@@ -5,8 +5,8 @@ on:
     branches:
       # A test branch for seeing if your tests will pass in your personal fork
       - test_me_github
-  pull_request_target:
-    types: [opened, reopened, synchronize]
+    #  pull_request_target:
```
Comment (trevor-vaughan) [inline .github/workflows/pr_acceptance.yml:8]: Can we uncomment this and try changing `ubuntu-latest` to `ubuntu-18.04` below?

It seems to be a bug in some of the underlying hosts and I've filed an [associated bug](https://github.community/t/podman-issues-on-ubuntu-latest/182129).

**Review by trevor-vaughan — DISMISSED** [review]
Let's try pinning to Ubuntu 18.04 in the acceptance tests => Never mind, this doesn't work

**Comment by trevor-vaughan** [comment]
This is passing in GitLab and we need to get it through to fix the GHA components. Approving and pushing.

**Review by trevor-vaughan — APPROVED** [review]

**Pull request merged**